packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.kt

```diff
@@ -66,6 +66,8 @@ class UpdatesModule(
         constants["isEnabled"] = updatesServiceLocal.configuration.isEnabled
         constants["releaseChannel"] = updatesServiceLocal.configuration.releaseChannel
         constants["isUsingEmbeddedAssets"] = updatesServiceLocal.isUsingEmbeddedAssets
+        constants["runtimeVersion"] = updatesServiceLocal.configuration.runtimeVersion ?: ""
+        constants["channel"] = updatesServiceLocal.configuration.requestHeaders["expo-channel-name"] ?: ""
```

Comment:
same comment as iOS and the catch block here (though I'm not clear on why https://github.com/expo/expo/pull/15451 wasn't an issue for Android 🤔 )

Reply:
The logic between the two platforms is pretty different. For example, iOS has three different possible returns, while android has a try/catch flow.

I re-orged a bit to make the flow more clear - #15451 didn't impact android as it always sets unless an error is thrown, while ios early returned is there was no `launchedUpdate`.